src/java/org/apache/cassandra/schema/TableMetadata.java

```diff
@@ -258,6 +264,7 @@ public Builder unbuild()
                .droppedColumns(droppedColumns)
                .indexes(indexes)
                .triggers(triggers)
+               .addConstraints(constraints)
```

Comment:
do we need to say "addConstraints"? What about just "constraints"? That would be aligned with all other methods on this builder.